buildscripts/grpc-java-artifacts-aarch64/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM dockcross/manylinux2014-aarch64
```

Comment:
Please no. It seems like this is turtles all the way down and strongly Python-centric.

Can we just add the following to our existing CentOS 7 docker file (`buildscripts/grpc-java-artifacts/Dockerfile`)?
```
yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
yum install -y gcc-c++-aarch64-linux-gnu
```

Reply:
I undestand the concern. the problem though is that after you install `gcc-c++-aarch64-linux-gnu` in centos 7 you still don't have the corresponding libc and libstdc++ for aarch64 and there seems to be no packages for that (the gcc package itself says it's for "kernel compilation only").  So setting up a crosscompiler to work properly in centos7 doesn't seem trivial (or maybe I'm missing something?). Perhaps that's also the reason why in the original version of code  you're actually running `apt-get install -y g++-aarch64-linux-gnu` directly on the ubuntu kokoro worker (and not in the grpc-java-artifacts image).

I could base the image off of debian (where crosscompiler is more readily available), but then the binary compatibility of the resulting binary wouldn't be that good I think.

The reason why I was using `dockcross/manylinux2014-aarch64` is that it's CentOS7 based, it has a working crosscompiler and it's purpose is to produce widely compatible binaries (and we want that).
If you are aware of a good centos7 base image with a crosscompiler, please let me know.

For now I'm keeping `dockcross/manylinux2014-aarch64` since the other solution doesn't work. Please let me know how you want me to handle this.

Reply:
grpc-java-artifacts Docker image was originally to get CentOS 5 environment (but recently it became CentOS 6). There was no need for that environment with aarch64. As far as I'm concerned, we don't have to run in a docker image and even if we did, debian-based would be fine. Since we statically link the binaries only the glibc version matters. But we don't need to figure that out now because I agree that this is biting off too much right now.